examples/GAN/gan_mnist_pytorch/README.md

```diff
@@ -0,0 +1,16 @@
+
+*This folder contains the required files and the example code to demonstrate how to train a simple GAN on the MNIST dataset.*
+## The file version can be found on this [PyTorch Lightning GAN example](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/domain_templates/generative_adversarial_net.py)
+
+### Folders and Files
+* **model_def.py**: Contains the core code for the model. This includes building and compiling the model.
+* **data.py**: Contains the data loading and preparation code for the model.
+* **const.yaml**: Contains the basic configuration for the experiment. 
```

Comment:
"A configuration file that trains the model with constant hyperparameter values."